libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c

```diff
@@ -798,7 +799,7 @@ TEST( HTTPS_Client_Unit_API, AddHeaderInvalidParameters )
     TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
     returnCode = IotHttpsClient_AddHeader( reqHandle, HTTPS_HOST_HEADER, FAST_MACRO_STRLEN( HTTPS_HOST_HEADER ), HTTPS_TEST_ADDRESS, FAST_MACRO_STRLEN( HTTPS_TEST_ADDRESS ) );
     TEST_ASSERT_EQUAL( IOT_HTTPS_INVALID_PARAMETER, returnCode );
-    returnCode = IotHttpsClient_AddHeader( reqHandle, HTTPS_CONTENT_LENGTH_HEADER, FAST_MACRO_STRLEN( HTTPS_CONTENT_LENGTH_HEADER ), pTestContentLengthValueStr, FAST_MACRO_STRLEN( pTestContentLengthValueStr ) );
+    returnCode = IotHttpsClient_AddHeader( reqHandle, HTTPS_CONTENT_LENGTH_HEADER, FAST_MACRO_STRLEN( HTTPS_CONTENT_LENGTH_HEADER ), pTestContentLengthValueStr, strlen( pTestContentLengthValueStr ) );
```

Comment:
Why this change?

Reply:
This was the warning associated with using the macro on a `char *`:
```
      |                                            ~~~~~~~~~^~~~~~
In file included from /home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/standard/https/test/unit/iot_tests_https_common.h:43,
                 from /home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c:31:
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c: In function 'TEST_HTTPS_Client_Unit_API_AddHeaderInvalidParameters_':
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/standard/https/src/private/iot_https_internal.h:256:51: warning: division 'sizeof (char *) / sizeof (char)' does not compute the number of array elements [-Wsizeof-pointer-div]
  256 | #define FAST_MACRO_STRLEN( x )    ( ( sizeof( x ) / sizeof( char ) ) - 1 )
      |                                                   ^
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c:801:162: note: in expansion of macro 'FAST_MACRO_STRLEN'
  801 |     returnCode = IotHttpsClient_AddHeader( reqHandle, HTTPS_CONTENT_LENGTH_HEADER, FAST_MACRO_STRLEN( HTTPS_CONTENT_LENGTH_HEADER ), pTestContentLengthValueStr, FAST_MACRO_STRLEN( pTestContentLengthValueStr ) );
      |                                                                                                                                                                  ^~~~~~~~~~~~~~~~~
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/standard/https/test/unit/iot_tests_https_client.c:772:12: note: first 'sizeof' operand was declared here
  772 |     char * pTestContentLengthValueStr = "0";
```

Reply:
Basically, the operation `pTestContentLengthValueStr ` is of `char *` and the compiler warning is against use of non-array type (`sizeof( char *)` operation) for determining size of array